.github/workflows/ci.yml

```diff
@@ -102,7 +120,7 @@ jobs:
 
 
   build-test-apps:
-    needs: [check-format, build-test-web, prepare-json-files, check-types]
+    needs: [prepare-json-files]
```

Comment:
Reduce deps here, those can run in parallel. publish-apps now depends on all of these tho